src/inference/rejection.js

```diff
@@ -22,6 +21,17 @@ module.exports = function(env) {
       maxScore: 0,
       incremental: false
     });
+
+    this.throwOnError = options.throwOnError !== undefined ? options.throwOnError : true;
+    // the value of options.probe is the min sample rate
+    this.probe = options.probe;
+    if (this.probe) {
+      // not throw error in probe mode
+      this.throwOnError = false;
```

Comment:
I think `DefaultInfer` should pass in `{throwOnError: false}` rather than assuming that when we 'probe' we'll always want this particular error behavior.